src/controllers/index.controller.js

```diff
@@ -12,7 +12,8 @@ const { notificationBus } = require('../services/notification.service');
 const { cantSendTxErrorMessage, API_ERROR_CODES } = require('../helpers/constants');
 const { parametersValidation } = require('../helpers/validations.helper');
 const { sanitizeLogInput } = require('../logger');
-const { getReadonlyWalletConfig, getWalletConfigFromSeed, WalletStartError } = require('../helpers/wallet.helper');
+const { getReadonlyWalletConfig, getWalletConfigFromSeed } = require('../helpers/wallet.helper');
+const { WalletStartError } = require('../errors');
```

Comment:
nitpick: I'm curious: was this import incorrect?

Reply:
Yeah, this file does not export `WalletStartError`